poly-synth/src/poly-synth.js

```diff
@@ -0,0 +1,114 @@
+/**
+ * Copyright 2017 Google Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * @class PolySynth
+ * @description Manages the life cycle of voices.
+ */
+class PolySynth {
+  /**
+   * @constructor
+   * @param {AudioContext} context The audio context.
+   */
+  constructor(context) {
+    if (!(context instanceof AudioContext))
+      throw context + ' is not a valid audio context.';
+
+    this.context_ = context;
+    this.activeVoices_ = [];
+    this.releasedVoices_ = [];
+
+    // The default maximum cutoff is set below the upper edge of human
+    // hearing and the minimum cutoff is above the lower edge of human hearing.
+    // Frequencies at either extreme are not typically used in electronic music.
```

Comment:
This comment is also wrong. You can just say "both frequencies are experimentally determined" because you will tweak them in the later commits. Right?